contracts/misc/ParaSpaceRaffle.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity ^0.8.10;
+
+import {VRFV2WrapperConsumerBase} from "../dependencies/chainlink/VRFV2WrapperConsumerBase.sol";
+import {LinkTokenInterface} from "../dependencies/chainlink/interfaces/LinkTokenInterface.sol";
+import {MerkleProof} from "../dependencies/openzeppelin/contracts/MerkleProof.sol";
+import {Ownable} from "../dependencies/openzeppelin/contracts/Ownable.sol";
+
+contract ParaSpaceRaffle is VRFV2WrapperConsumerBase, Ownable {
```

Comment:
can we use ACLManager instead to avoid managing so many owners

Reply:
this is a one time contract. no need for that